### DISCUSSION TITLE
Unable to connect to server with my domain name in the URL?

### DISCUSSION THREAD

**Discussion by trustieee** [datalust/seq-tickets#1147] [Help]
Seq config:

```
    "listenUris": [
      "http://localhost:5341",
      "https://seq-uat.domain.com",
      "https://localhost:45341"
    ],
```
I can successfully browse to http://localhost:5341.
If I browse to https://localhost:45341 then I receive a browser error about TLS/Cypher issues, which is fine.
If I browse to https://seq-uat.domain.com then I receive a page can't be displayed browser message.

I've added my domain.com certificate to the store, and used `seq bind-ssl --thumbprint="my thumbprint here"` successfully. What else could I be missing?

**Comment by nblumhardt** [comment]
Hi! Could DNS lookup be failing? If you run:

```
nslookup seq-uat.domain.com
```

do you get a successful result?
